jac/jaclang/compiler/parser.py

```diff
@@ -611,8 +612,11 @@ def decorators(self, kid: list[ast.AstNode]) -> ast.SubNodeList[ast.Expr]:
 
             decorators: (DECOR_OP atomic_chain)+
             """
-            valid_decors = [i for i in kid if isinstance(i, ast.Expr)]
-            if len(valid_decors) == len(kid) / 2:
+            valid_decors = []
```

Comment:
Remove this list here and bellow, we don't validate like this, `match` and `consume` will do the job